Source/WebKit/WebProcess/Network/WebLoaderStrategy.cpp

```diff
@@ -1073,6 +1073,63 @@ void WebLoaderStrategy::prioritizeResourceLoads(const Vector<WebCore::Subresourc
     connection.send(Messages::NetworkConnectionToWebProcess::PrioritizeResourceLoads(identifiers), 0);
 }
 
+ResourceError WebLoaderStrategy::cancelledError(const ResourceRequest& request) const
+{
+    return WebKit::cancelledError(request);
```

Comment:
I think this is in the WebKit namespace and therefore we shouldn't need the prefix.

Reply:
We do need the WebKit:: because otherwise it thinks it's recursion.